### ISSUE TITLE
timer-log-main example: The example is not working in dev mode

### ISSUE THREAD

**Issue by aldettinger** [apache/camel-quarkus#2024]
Especially, the step [Start in the Development mode](https://github.com/apache/camel-quarkus-examples/tree/master/timer-log-main#start-in-the-development-mode) propose to execute the command below:
`mvn clean compile quarkus:dev -Dquarkus.args='Hello 2'`

However, an ERROR log is produced:
```
2020-11-24 13:37:04,373 ERROR [io.qua.run.boo.StartupActionImpl] (Quarkus Main Thread) Error running Quarkus: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.quarkus.runner.bootstrap.StartupActionImpl$3.run(StartupActionImpl.java:134)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.IllegalStateException: Expected at least two CLI arguments
	at org.acme.timer.Main.main(Main.java:47)
	... 6 more

2020-11-24 13:37:04,414 INFO  [io.qua.dep.dev.IsolatedDevModeMain] (main) Attempting to start hot replacement endpoint to recover from previous Quarkus startup failure
```

**Comment by aldettinger** [comment]
Opened [ an upstream quarkus issue #13451](https://github.com/quarkusio/quarkus/issues/13451).

**Comment by aldettinger** [comment]
This is a quarkus regression, it's fixed with quarkus >= 1.10.2.Final.

**Comment by aldettinger** [comment]
Fixed in https://github.com/apache/camel-quarkus-examples/commit/762260f7bd16aa3e8c996a98dacc2a4fe10b08ee.
